telegram/utils/request.py

```diff
@@ -279,35 +316,41 @@ def post(self, url, data, timeout=None):
 
         return self._parse(result)
 
-    def retrieve(self, url, timeout=None):
+    def retrieve(self, url, timeout=None, retries=None):
         """Retrieve the contents of a file by its URL.
 
         Args:
             url (:obj:`str`): The web location we want to retrieve.
             timeout (:obj:`int` | :obj:`float`): If this value is specified, use it as the read
                 timeout from the server (instead of the one specified during creation of the
                 connection pool).
-
+            retries (:obj:`int`): If this value is specified, use it as the number of retries for
+                the request (instead of the one specified during creation of the connection pool).
         """
         urlopen_kwargs = {}
         if timeout is not None:
             urlopen_kwargs['timeout'] = Timeout(read=timeout, connect=self._connect_timeout)
 
+        if retries is not None:
+            urlopen_kwargs['retries'] = self._parse_retries(retries)
+
         return self._request_wrapper('GET', url, **urlopen_kwargs)
 
-    def download(self, url, filename, timeout=None):
+    def download(self, url, filename, timeout=None, retries=None):
         """Download a file by its URL.
 
         Args:
             url (str): The web location we want to retrieve.
             timeout (:obj:`int` | :obj:`float`): If this value is specified, use it as the read
                 timeout from the server (instead of the one specified during creation of the
                 connection pool).
+            retries (:obj:`int`): If this value is specified, use it as the number of retries for
+                the request (instead of the one specified during creation of the connection pool).
 
           filename:
```

Comment:
ehm. if we touch the docstring here it looks like we can fix it a bit.
would you mind moving `filename` after `url` and fix the alignment for all args to be like `filename`'s?